docs/zh_CN/CommunitySharings/HpoComparison.md

```diff
@@ -0,0 +1,221 @@
+# 超参数优化的对比
```

Comment:
refer to guidance to remove deleted files.

Reply:
fixed.